### PR TITLE
Update ring & untrusted

### PR DESCRIPTION
**Pull request by serprex**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -15,11 +15,11 @@ rustls-pemfile = "2"
 pki-types = { package = "rustls-pki-types", version = "1" }
 
 [dev-dependencies]
-ring = "0.16.5"
+ring = "0.17"
 rustls = "0.22"
 rustls-webpki = "0.102"
 serial_test = "2"
-untrusted = "0.7" # stick to the version ring depends on for now
+untrusted = "0.9" # stick to the version ring depends on for now
```
Comment (djc) [inline Cargo.toml:22]: I think this comment is now no longer relevant, would you mind removing it?

**Review by djc — APPROVED** [review]
Thanks!

File: Cargo.toml (hunk 1 above)
Comment (serprex) [reply Cargo.toml:22]: I left it since I updated it to the exact version ring depends on

**Review by cpu — APPROVED** [review]
Thanks :+1: Might as well keep the dev deps up to date too.

**Comment by cpu** [comment]
> cpu force-pushed the main branch from 427fa44 to 6019ccb 

Squashed the commit history.

**Pull request merged**